src/App.jsx

```diff
@@ -83,51 +90,60 @@ function App() {
     });
   });
 
+  onCleanup(() => {
+    window.removeEventListener("resize", handleWindowResize);
+  });
+
   return (
-    <div
-      class={classNames(styles.App, {
-        "theme-light": is_light_theme(),
-        "theme-dark": !is_light_theme(),
-      })}
-    >
-      <Show when={banner_message()}>
-        <BannerComponent
-          message={banner_message()}
-          category={banner_category.ERROR}
-          showCloseButton
-        />
-      </Show>
-      <NavBar />
-      <section
-        class={classNames(styles.content, {
-          [styles["is-acc-switcher-open"]]: showAccountSwitcher(),
+    <>
```

Comment:
Rather than using am empty tag, may be we can use `Show` with `when` condition and `fallback` for rendering Body or`EmptyView`

Reply:
emptyview is just an overlay to block the user from performing any actions in the app. so if given as a fallback, won't the app rerender when the view is supported initially, then unsupported and then supported again. so in that case since we don't want to interfere with the rendering shouldn't it given the way it is now?